lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -60,6 +60,12 @@ div.course-wrapper {
       }
     }
 
+    #sr-is-focusable,
```

Comment:
Yeah, let me do some research. A class might be better here, especially if we're only sending focus to it. No need for the ID.